apps/system/index.html

```diff
@@ -344,6 +346,8 @@
       <!-- XXX: Before we can make LockScreen as a iframe or app, we need this to split LockScreen and System part in this way (Bug 1057198) -->
       <div id="lockscreen-frame-placeholder"></div>
 
+      <div id="fullscreen-overlay" data-z-index-level="fullscreen-overlay"></div>
+
```

Comment:
I don't like the idea that we are adding something to DOM that won't be used for most of the life of the phone. Let's move this overlay and it's associated styles completely into the Notice module.